### ISSUE TITLE
How to reproduce your results

### ISSUE THREAD

**Issue by yangjiangeyjg** [LeonHLJ/FAC-Net#4]
The first three lines are the result of my own training model：
**100 100 34.18 28.70 20.20 13.54 8.17 3.40 1.45
35 70 31.39 26.72 19.16 12.83 7.28 3.66 1.70
35 75 31.38 26.49 19.76 13.25 7.76 3.84 1.81**
yours:
**1000 1000 68.19 62.43 53.21 44.78 34.10 23.13 13.44**

**Comment by LeonHLJ** [comment]
> The first three lines are the result of my own training model：
> **100 100 34.18 28.70 20.20 13.54 8.17 3.40 1.45 35 70 31.39 26.72 19.16 12.83 7.28 3.66 1.70 35 75 31.38 26.49 19.76 13.25 7.76 3.84 1.81**
> yours:
> **1000 1000 68.19 62.43 53.21 44.78 34.10 23.13 13.44**

Our training settings are all presented in train_options.py, and there is no training trick in our method, you just need to train our model from scratch. Did you change some hyper-parameters?

**Comment by yangjiangeyjg** [comment]
> 
> 
> > The first three lines are the result of my own training model：
> > **100 100 34.18 28.70 20.20 13.54 8.17 3.40 1.45 35 70 31.39 26.72 19.16 12.83 7.28 3.66 1.70 35 75 31.38 26.49 19.76 13.25 7.76 3.84 1.81**
> > yours:
> > **1000 1000 68.19 62.43 53.21 44.78 34.10 23.13 13.44**
> 
> Our training settings are all presented in train_options.py, and there is no training trick in our method, you just need to train our model from scratch. Did you change some hyper-parameters?

I didn't change any hyper-parameters. Is the dataset Thumos14reduced-I3D-JOINTFeatures?

**Comment by LeonHLJ** [comment]
> > > The first three lines are the result of my own training model：
> > > **100 100 34.18 28.70 20.20 13.54 8.17 3.40 1.45 35 70 31.39 26.72 19.16 12.83 7.28 3.66 1.70 35 75 31.38 26.49 19.76 13.25 7.76 3.84 1.81**
> > > yours:
> > > **1000 1000 68.19 62.43 53.21 44.78 34.10 23.13 13.44**
> > 
> > 
> > Our training settings are all presented in train_options.py, and there is no training trick in our method, you just need to train our model from scratch. Did you change some hyper-parameters?
> 
> I didn't change any hyper-parameters. Is the dataset Thumos14reduced-I3D-JOINTFeatures?

Yes, I noticed the last line's result is the same as ours, so I think your features and annotations are right. Maybe you can change the random seed and train again. On our server, for the flow stream, the mAP at IoU 0.1 exceeds 50% after 10 epochs. Hope this information could be helpful to you.

**Comment by yangjiangeyjg** [comment]
> 
> 
> > > > The first three lines are the result of my own training model：
> > > > **100 100 34.18 28.70 20.20 13.54 8.17 3.40 1.45 35 70 31.39 26.72 19.16 12.83 7.28 3.66 1.70 35 75 31.38 26.49 19.76 13.25 7.76 3.84 1.81**
> > > > yours:
> > > > **1000 1000 68.19 62.43 53.21 44.78 34.10 23.13 13.44**
> > > 
> > > 
> > > Our training settings are all presented in train_options.py, and there is no training trick in our method, you just need to train our model from scratch. Did you change some hyper-parameters?
> > 
> > 
> > I didn't change any hyper-parameters. Is the dataset Thumos14reduced-I3D-JOINTFeatures?
> 
> Yes, I noticed the last line's result is the same as ours, so I think your features and annotations are right. Maybe you can change the random seed and train again. On our server, for the flow stream, the mAP at IoU 0.1 exceeds 50% after 10 epochs. Hope this information could be helpful to you.



Thanks for your advice. I didn't realize that random seed is so important for small model.
